### PR TITLE
cryptocom: update safeNetwork

### PR DESCRIPTION
**Pull request by sc0Vu**

### CODE REVIEW

File: js/cryptocom.js [hunk 1]
```diff
@@ -1422,9 +1423,16 @@ module.exports = class cryptocom extends Exchange {
     }
 
     safeNetwork (networkId) {
-        // stub for now
-        // TODO: figure out which networks are supported on cryptocom
-        return networkId;
+        const networksById = {
+            'BTC': 'BTC',
```
Comment (carlosmiei) [inline js/cryptocom.js:1427]: question, what's the benefit of having this map if we're going to map X to X? We're using the same id for the key and value

Comment (sc0Vu) [reply js/cryptocom.js:1427]: Do we have unify network name, eg ERC20?

I was thinking to add `ERC20: ETH` or `BEP20: BNB` in safeNetwork, and we can use that when user call withdraw.

Comment (carlosmiei) [reply js/cryptocom.js:1427]: @sc0Vu I think the network unification is not "complete" yet, but yes we usually convert network ids to ERC20, TRC20, BSC,OMNI,SOL,BCH, etc (you can check other `safeNetwork` Implementations for more examples).

My point is if the name is the same we don't need to list it right? Just the different ones

Comment (sc0Vu) [reply js/cryptocom.js:1427]: For now it's only for clarity, we can leave this PR till  the network unification is completed.

**Review by carlosmiei — APPROVED** [review]

**Review by carlosmiei — APPROVED** [review]

**Pull request merged**